slack.go

```diff
@@ -49,11 +40,29 @@ func (s *Slack) StartRealTimeMessagingListener(ctx context.Context) {
 		case *slack.RTMError:
 			log.Printf("Error received on RTM channel: %v", event.Error())
 		case *slack.DisconnectedEvent:
-			log.Print("Disconnected from RTM channel")
-			return
+			if !*standupIsOver {
```

Comment:
This is what the context above is for. `if !ctx.Done() { //reconnect // }`

https://golang.org/pkg/context/

You should be able to do this without passing around the magic boolean pointer